_harp/academy/_about.md

```diff
@@ -2,4 +2,18 @@
 
 <img src="/assets/fac-logo.png" class="fac-logo">
 
-Founders & Coders runs an intensive full-time, 16-week course in full-stack software development using JavaScript and Node.js. Application is highly-selective and we take just 16 students per cohort. After graduation, we expect all of our students to help us deliver the course to the next cohort and to contribute to our various community projects.
+Founders & Coders has created a new model for education.
+
+We run a full-time, 16-week course in full-stack software development using JavaScript and Node.js. 46/47 of our recent graduates are working as software developers and around 40% of our graduates are female.
+
+We aim to give students the best possible training in software development. In return we ask you to help us deliver subsequent courses and to contribute to our various community projects.
+
+We also ask you to contribute in one of three ways:
+
+- Join our community - earning a living while learning and working on interesting projects with our senior developers.
+
+- Let us to place you with one of our partners so that we can get a recruitment fee.
+
+- Donate 10% of your salary for your first year of employment.
```

Comment:
I made this clear in my head but nowhere else. What I meant to say is that: If you don't want to stay with us or "let us help you find full time employment" and instead want to find your own job then you might want to consider donating 10% of your first year salary to FAC. It's not something we've asked before but as I said in the comment it might work well for people who want to work outside of London or who find a job with a friend, recruiter or company that doesn't pay recruitment fees. What do you think?